Client/UITableView+Extension.swift

```diff
@@ -0,0 +1,37 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+// MARK: - Visible Headers
+extension UITableView {
```

Comment:
This was in the class as private since this is the only place it's used, and this code should frankly not be used anywhere else. I would keep it as private in the file to discourage anyone from using it elsewhere (if/when we refactor the menu, another method should be used and this code will disappear).

Reply:
Ah, then let's make it private 🙈